ui/imports/shared/views/chat/InvitationBubbleView.qml

```diff
@@ -69,158 +70,144 @@ Item {
-                        color: d.invitedCommunity.verifed ? Theme.palette.primaryColor1 : Theme.palette.baseColor1
-                        font.weight: Font.Medium
-                        font.pixelSize: 13
-                    }
-
-                    StatusBaseText {
-                        id: invitedYou
-
-                        Layout.fillWidth: true
-
-                        visible: text != ""
-                        text: {
-                            // Not Refactored Yet
-                            return ""
-    //                        if (root.store.chatsModelInst.channelView.activeChannel.chatType === Constants.chatType.oneToOne) {
-    //                            return isCurrentUser ?
-    //                                        qsTr("You invited %1 to join a community").arg(root.store.chatsModelInst.userNameOrAlias(root.store.chatsModelInst.channelView.activeChannel.id))
-    //                                      : qsTr("%1 invited you to join a community").arg(displayUserName)
-    //                        } else {
-    //                            return isCurrentUser ?
-    //                                        qsTr("You shared a community")
-    //                                      : qsTr("A community has been shared")
-    //                        }
-                        }
-                        wrapMode: Text.WordWrap
-                    }
+            spacing: 4
+
+            StatusBaseText {
+                id: title
+
+                Layout.fillWidth: true
+
+                text: d.invitedCommunity.verifed ? qsTr("Verified community invitation") : qsTr("Community invitation")
+                color: d.invitedCommunity.verifed ? Theme.palette.primaryColor1 : Theme.palette.baseColor1
+                font.weight: Font.Medium
+                font.pixelSize: 13
+            }
+
+            StatusBaseText {
+                id: invitedYou
+
+                Layout.fillWidth: true
+
+                visible: text != ""
+                text: {
+                    // Not Refactored Yet
+                    return ""
+//                    if (root.store.chatsModelInst.channelView.activeChannel.chatType === Constants.chatType.oneToOne) {
+//                        return isCurrentUser ?
+//                                    qsTr("You invited %1 to join a community").arg(root.store.chatsModelInst.userNameOrAlias(root.store.chatsModelInst.channelView.activeChannel.id))
+//                                  : qsTr("%1 invited you to join a community").arg(displayUserName)
+//                    } else {
+//                        return isCurrentUser ?
+//                                    qsTr("You shared a community")
+//                                  : qsTr("A community has been shared")
+//                    }
                 }
+                wrapMode: Text.WordWrap
+            }
```

Comment:
I think this text can be removed completely. We no longer do "invites"

Reply:
Good point, will remove

Reply:
Done